icaruscode/Filters/FilterCRTPMTMatching_module.cc

```diff
@@ -347,56 +351,42 @@ icarus::crt::FilterCRTPMTMatching::FilterCRTPMTMatching(
   }
 }
 
-void icarus::crt::FilterCRTPMTMatching::getTriggerConf(art::Run& r) {
+//void icarus::crt::FilterCRTPMTMatching::getTriggerConf(art::Run const& r) {
+//  fTriggerConfiguration =
+//      r.getProduct<icarus::TriggerConfiguration>(fTriggerConfigurationLabel);
+//}
+
+void icarus::crt::FilterCRTPMTMatching::beginRun(art::Run const& r) {
```

Comment:
```suggestion
void icarus::crt::FilterCRTPMTMatching::beginRun(art::Run& r) {
```
In principle, in `beginRun()` of a producer or filter, you _could_ add data products to the run, and that's the reason the argument is not declared `const`. You need to replicate the exact signature in order for _art_ to properly call it.
In that sense, the advantage of `getTriggerConf(art::Run const& r)` was that you were not imposed any signature and you could declare `r` as `const`. In this respect, having `beginRun()` call `getTriggerConf()` has still a reason, but I am not endorsing either way over the other here.